client/src/Components/App/App.js

```diff
@@ -18,7 +23,11 @@ const App = () => {
 				<div className = 'd-flex flex-column align-items-center col-9 align-self-start'>
 					{posts ?
 						(
-							posts.map((d, idx) => { return <Post key = {idx} text = {d.text}/>; })
+							posts.filter((post) => {
+								return typeof post.title == 'string' && typeof post.body == 'string';
+							}).map((post, idx) => { 
+								return <Post key = {post._id} title = {post.title} body = {post.body}/>; 
+							})
```

Comment:
Sorry, I missed the point. Why do we need this filter?

Reply:
React errors out if the props passed to a component are javascript objects. I thought it would be good to include it just in case

Reply:
I will remove it though and just have this validation be done on the server